estate/models/estate_property_tags.py

```diff
@@ -0,0 +1,7 @@
+from odoo import fields, models
+
+class EstatePropertyTage(models.Model):
+    _name = "estate.property.tage"
```

Comment:
Tags instead of tage ? could provoke some headaches later on, better change it ASAP :)